data/categories.txt

```diff
@@ -6093,67 +6093,651 @@ sw:Bwawa la kuogelea
+de:Amerikanischer Fußball
+el:αμερικάνικο ποδόσφαιρο
+es:Fútbol americano
+es-MX:Fútbol americano
+eu:Futbol amerikarra
+fa:فوتبال آمریکایی
+fi:Amerikkalainen jalkapallo
+fr:Football américain
+he:כדורגל אמריקאי
+hu:Amerikai foci
+id:Sepak Bola Amerika
+it:Football americano
+ja:アメリカンフットボール
+ko:미식 축구
+mr:अमेरिकन फुटबॉल
+nb:Amerikansk fotball
+nl:Amerikaans voetbal
+pl:Futbol amerykański
+pt:Futebol americano
+pt-BR:Futebol americano
+ro:Fotbal american
+ru:Aмериканский футбол
+sk:Americký futbal
+sv:Amerikansk fotboll
+sw:Soka ya Marekani
+th:อเมริกันฟุตบอล
+tr:Amerikan futbolu
+uk:Американський футбол
+vi:Bóng bầu dục Mỹ
+zh-Hans:美式足球
+zh-Hant:美式足球
+
+sport-archery
+en:Archery
+ar:الرماية
+be:Стральба з лука
+cs:Lukostřelba
+da:Bueskydning
+de:Bogenschießen
+el:Τοξοβολία
+es:Tiro al arco
+es-MX:Tiro al arco
+eu:Arku-tiroa
+fa:تیراندازی با کمان
+fi:Jousiammunta
+fr:Tir à l'arc
+he:קַשׁתוּת
+hu:Íjászat
+id:Panahan
+it:Tiro con l'arco
+ja:アーチェリー
+ko:양궁
+mr:धनुर्विद्या
+nb:Bueskyting
+nl:Boogschieten
+pl:Łucznictwo
+pt:Tiro com arco
+pt-BR:Tiro com arco
+ro:TIR cu arcul
+ru:Стрельба из лука
+sk:Lukostreľba
+sv:Bågskytte
+sw:Upigaji mishale
+th:ยิงธนู
+tr:Okçuluk
+uk:Стрільба з лука
+vi:Bắn cung
+zh-Hans:射箭
+zh-Hant:射箭
+
+sport-athletics
+en:Athletics
+ar:ألعاب القوى
+be:Лёгкая атлетыка
+bg:атклетика|лека
+cs:Atletika
+da:Atletik
+de:Leichtathletik
+el:Στίβος
+es:Atletismo
+eu:Atletismoa
+fa:ورزشکاری
+fi:Yleisurheilu
+fr:Athlétisme
+he:אַתלֵטִיקָה
+hu:Atlétika
+id:Atletik
+it:Atletica leggera
+ja:陸上競技
+ko:체육시설
+mr:शरीरिक कसरती|ऍथलेटिक्स
+nb:Friidrett
+nl:Atletiek
+pl:Lekkoatletyka
+pt-BR:Atletismo
+pt:Atletismo
+ro:Atletism
+ru:Лёгкая атлетика
+tr:Atletik
+uk:Легка атлетика
+vi:Thế vận hội
+zh-Hans:竞技
+zh-Hant:競技
+
+sport-australian_football
+en:Australian Football
+ar:كرة القدم الأسترالية
+be:Аўстралійскі футбол
+bg:Австралийски футбол
+cs:Australský fotbal
+da:Australsk fodbold
+de:Australian Football
+el:Αυστραλιανό ποδόσφαιρο
+es:Fútbol australiano
+eu:Australiako futbol araua
+fa:فوتبال استرالیایی
+fi:Australialainen jalkapallo
+fr:Football australien
+he:פוטבול אוסטרלי
+hu:Ausztrál futball
+id:Sepak bola Australia
+it:Football australiano
+ja:オーストラリアンフットボール
+ko:오스트레일리안 풋볼
+mr:रग्बी|ऑस्ट्रेलियन फुटबॉल
+nb:australsk fotball
+nl:Australian football
+pl: Futbol australijski
+pt-BR:Futebol australiano
+pt:Futebol australiano
+ro:Fotbal australian
+ru: Австралийский футбол
+tr:Avustralya futbolu
+uk:Австралійський футбол
+vi:Bóng bầu dục Úc
+zh-Hans:澳式足球
+zh-Hant:澳式足球
+
+sport-baseball
+en:Baseball
+ar:البيسبول
+be:Бейсбол
+cs:Baseball
+da:Baseball
+de:Baseball
+el:Μπέιζμπολ
+es:Béisbol
+es-MX:Béisbol
+eu:Beisbol
+fa:بیسبال
+fi:Baseball
+fr:Base-ball
+he:בייסבול
+hu:Baseball
+id:Baseball
+it:Baseball
+ja:野球
+ko:야구
+mr:बेसबॉल
+nb:Baseball
+nl:Basketbal
+pl:Baseball
+pt:Beisebol
+pt-BR:Beisebol
+ro:Baseball
+ru:Бейсбол
+sk:Baseball
+sv:Baseboll
+sw:Baseball
+th:เบสบอล
+tr:Beyzbol
+uk:бейсбол
+vi:Bóng chày
+zh-Hans:棒球
+zh-Hant:棒球
 
 sport-basketball
```

Comment:
@vng @pastk are these new and old types added into the generalized category like "sport(s)"?

Reply:
there is no such category yet

Reply:
@vng can you add it?

Reply:
Can be done in this PR, the same as @food or any other in this file.

Reply:
What will be a use case for such a category though?
I can see people searching for e.g. a sports venue or a sports ground / fitness station around, but why looking for a list of kinds of sports?